jujugui/static/gui/src/app/init/component-renderers-mixin.js

```diff
@@ -351,6 +353,31 @@ const ComponentRenderersMixin = (superclass) => class extends superclass {
   }
 
   /**
+    Opens the help overlay.
+  */
+  _renderHelp(state, next) {
+    ReactDOM.render(<Help
+      changeState={this.state.changeState.bind(this.state)}
```

Comment:
see `_bindRenderUtilities` in init.js for a pre-bound version of this.

Reply:
done